src/CGRANode.cpp

```diff
@@ -190,17 +196,25 @@ bool CGRANode::canSupport(DFGNode* t_opt) {
       (t_opt->isReturn()     and !canReturn()) or
       (t_opt->isCall()       and !canCall())  or
       (t_opt->isVectorized() and !supportVectorization()) or
-      (t_opt->hasCombined()  and !supportComplex()) or
+      // (t_opt->hasCombinedExceptMem()  and !supportComplex()) or
       (t_opt->isAdd()        and !canAdd()) or 
       (t_opt->isMul()        and !canMul()) or 
       (t_opt->isPhi()        and !canPhi()) or 
       (t_opt->isSel()        and !canSel()) or 
-      (t_opt->isMAC()        and !canMAC()) or 
+      // (t_opt->isMAC()        and !canMAC()) or 
```

Comment:
So `MAC` is not a basic op any more, how would user support it then?

Reply:
I uncommentted it.